examples/streaming-example/src/main/java/io/getunleash/example/StreamingExample.java

```diff
@@ -0,0 +1,65 @@
+package io.getunleash.example;
+
+import io.getunleash.DefaultUnleash;
+import io.getunleash.Unleash;
+import io.getunleash.UnleashException;
+import io.getunleash.event.ClientFeaturesResponse;
+import io.getunleash.event.UnleashSubscriber;
+import io.getunleash.util.UnleashConfig;
+
+public class StreamingExample {
+
+    private static Unleash unleash;
+
+    private static class StreamingEventSubscriber implements UnleashSubscriber {
+        @Override
+        public void togglesFetched(ClientFeaturesResponse toggleResponse) {
+            System.out.println("[STREAMING EVENT] Features updated");
+            if (unleash != null) {
+                boolean isEnabled = unleash.isEnabled("streaming_flag");
+                System.out.println("  streaming_flag: " + (isEnabled ? "ENABLED" : "DISABLED"));
+            }
+        }
+
+        @Override
+        public void onError(UnleashException unleashException) {
+            System.err.println("[STREAMING ERROR] " + unleashException.getMessage());
+        }
+    }
+
+    public static void main(String[] args) {
+        StreamingEventSubscriber subscriber = new StreamingEventSubscriber();
+
+        UnleashConfig config =
+                UnleashConfig.builder()
+                        .appName("streaming-example")
+                        .instanceId("streaming-example-instance")
+                        .unleashAPI(getOrElse("UNLEASH_API_URL", "https://app.unleash-hosted.com/demo/api"))
+                        .customHttpHeader(
+                                "Authorization",
+                                getOrElse("UNLEASH_API_TOKEN",
+                                        "*:development.25a06b75248528f8ca93ce179dcdd141aedfb632231e0d21fd8ff349"))
+                        .experimentalStreamingMode()
+                        .subscriber(subscriber)
+                        .build();
+
+        unleash = new DefaultUnleash(config);
+
+        System.out.println("Streaming client started. Waiting for events... (Press Ctrl+C to exit)");
+
+        try {
+            Thread.currentThread().join();
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            System.out.println("Streaming example shutting down");
+        }
+    }
```

Comment:
Looks ok, I'd like to also get some feature evaluation result, but I tested that manually and it works well.

I did something similar to this: https://github.com/Unleash/unleash-java-sdk/blob/41c6028287f707e1eac3d77d69eeb00f1039860a/examples/cli-example/src/main/java/io/getunleash/example/AdvancedConstraints.java#L29-L32 but limiting to 100 iterations with a sleep.

Reply:
There's `boolean isEnabled = unleash.isEnabled("streaming_flag");` which is a flag you need to create and you will see it changing instantly on every streaming change. This is a similar pattern I did in Node SDK where we react to instant changes instead of looping which was fine for the polling mindset but for streaming event listening mindset seems to be more appropriate for me.